widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -206,7 +206,7 @@ public void newEngagementLoaded(OmnicoreEngagement engagement) {
 
     public void initCall(String companyName,
                          String queueId,
-                         String contextUrl,
```

Comment:
Is it okay to drop support for contextUrl now in this public  function?

Reply:
It is kind of public but used only internally, integrator only deals with activities via launch intents.

Reply:
Previously we added backward compatibility for integrators who use this class manually.